load-test-framework/src/main/java/com/google/pubsub/flic/Driver.java

```diff
@@ -141,6 +167,11 @@
           "message loss. If set less than 1, this flag is ignored."
   )
   private int numberOfMessages = 0;
+  
+  private final String SPREADSHEET_ID = System.getenv("LOADTEST_SPREADSHEET_ID");
```

Comment:
That's what I had done originally, but it's a bit annoying putting in the value as a flag on every single run, and putting in our document as the default flag value would put the sheet id up publicly for all Github to see, which isn't too much of an issue I guess but I felt a bit weird about doing it.